packages/colorpicker/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: colorpicker
+description: "A new Flutter package project."
+version: 0.0.1
+publish_to: "none"
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=1.17.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  component_library:
+    path: ../component_library
```

Comment:
This package should be completely seperated from the other ones, because it will built seperately too and be integrated in another app which does not have the component library.
Please revert the changes and duplicate the checkerboard image.

Reply:
Actually I did that before but I was suggested to load the image from the `component_library` itself. Anyway, I'll revert it :)

Reply:
Yes I saw that, I'm sorry! 
Thank you! :)